topics/admin/tutorials/singularity/tutorial.md

```diff
@@ -141,11 +158,14 @@ First, we will install Singularity using Ansible.
 >    {: .code-out}
 {: .hands_on}
 
-## Configure Galaxy to use Singularity
+## Configure Galaxy to use Apptainer
 
-Now, we will configure Galaxy to run tools using Singularity containers, which will be automatically fetched from [the BioContainers repository](https://quay.io/organization/biocontainers).
+Now, we will configure Galaxy to run tools using Apptainer containers, which will be automatically fetched from [the BioContainers repository](https://quay.io/organization/biocontainers).  
 
-> <hands-on-title>Configure Galaxy to use Singularity</hands-on-title>
+{% icon galaxy-info %} Galaxy still uses `singularity` in most variables, they will be replaced successively.
```

Comment:
could you make this a 'tip' box since semantically it's a tip

Reply:
I didn't because I think it is very important that people read this before they are confused, but if you want I change this

Reply:
how about a warning box instead then?